### ISSUE TITLE
Ensure the coco endpoints exist for RFI system

### ISSUE THREAD

**Issue by jrs65** [kotekan/kotekan#508]
We need to ensure the following endpoints exist in kotekan and have them forwarded to from coco

- [x] bad-input-list
- [x] enable/disable zeroing
- [x] enable/disable SK + variance dump
- [x] change which feed variance is dumped for

**Comment by andrerenard** [comment]
The format of these endpoints will look like this:
```
updatable_config:
  bad_inputs:
    kotekan_update_endpoint: "json"
    # These inputs are assumed to be in cylinder order
    bad_inputs: []
  rfi_zeroing_toggle:
    kotekan_update_endpoint: "json"
    rfi_zeroing: False
  rfi_sk_record:
    kotekan_update_endpoint: "json"
    write_to: /mnt/gong/RFI/test_sk/
    write_to_disk: false
  rfi_var_record:
    kotekan_update_endpoint: "json"
    write_to: /mnt/gong/RFI/test_var/
    write_to_disk: false
  rfi_var_element_index:
    kotekan_update_endpoint: "json"
    # This element should be given in cylinder input order,
    # not correlator/FPGA input order.
    element_index: 840
```
The changes are being made in #528 and I'm just adding the last endpoint now.   I will open another ticket in the `coco` repo to add these to the `coco` config for the next run.   @nritsche let me know if you have questions about the function of these endpoints.

**Comment by andrerenard** [comment]
Cross reference: https://github.com/chime-experiment/coco/issues/169

**Comment by andrerenard** [comment]
Closed by #528 and corresponding `ch_ancible` changes.
